packages/cacti-plugin-copm-fabric/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@hyperledger-cacti/cacti-plugin-copm-fabric",
+  "version": "0.0.1",
```

Comment:
My bad, I missed this. 
@jenniferlianne Please see my earlier change requests and address them each if possible. If not possible for whatever reason then let's talk about it.

Reply:
I have updated the package.json dependencies to fixed versions as requested, and the copm packages to 2.0.0.  I thought your package version request for local packages only applied to published packages.  Using a tool like https://github.com/bmish/check-dependency-version-consistency could make enforcing dependency consistency easier.

Reply:
@jenniferlianne Cheers. Yeah that tool would be nice! It could reduce the amount of hand-holding PRs need in terms of package versioning, making reviews faster which makes everyone happier.